packages/synchro-charts/src/components/sc-dial/sc-dial-base/sc-dial-svg.tsx

```diff
@@ -0,0 +1,119 @@
+import { Component, h, Prop } from '@stencil/core';
+import { NO_VALUE_PRESENT } from '../../../constants';
+import { DataPoint, DataStream } from '../../../utils/dataTypes';
+import { round } from '../../../utils/number';
+import { getIcons } from '../../charts/common/annotations/iconUtils';
+import { Threshold } from '../../charts/common/types';
+import { DialSizeConfig } from '../type';
+import { sizeConfigurations } from './util';
+
+@Component({
+  tag: 'sc-dial-svg',
+  shadow: false,
+})
+export class ScDialSvg {
+  @Prop() percent: number;
+  @Prop() point?: DataPoint;
+  @Prop() breachedThreshold: Threshold;
+  @Prop() stream?: DataStream | null;
+  @Prop() size: DialSizeConfig;
+
+  render() {
+    const R = this.size.viewport / 2;
+    const r = R - this.size.dialThickness;
+    const perimeter = 2 * Math.PI * r;
+    const left = perimeter * (1 - this.percent);
+    const right = perimeter - left;
+    const labelColor = this.breachedThreshold?.color || sizeConfigurations.BLUE;
+    const label = this.breachedThreshold?.label?.show ? this.breachedThreshold?.label?.text : '';
+    const icon = this.breachedThreshold ? this.breachedThreshold.icon : undefined;
+    return (
+      <svg
+        width="100%"
+        height="100%"
+        viewBox={`0 0 ${this.size.viewport} ${this.size.viewport}`}
+        data-testid="current-value"
```

Comment:
is this actually used anywhere?

Reply:
Sorry, I'm not clear on your question.

Reply:
is it utilized in a test? why do we have this attribute present?

Reply:
Yes, in cypress/integration/dial/dial.spec.ts